scripts/setup/generate_secrets.py

```diff
@@ -102,6 +103,11 @@ def add_secret(name: str, value: str) -> None:
     if development and need_secret("local_database_password"):
         add_secret("local_database_password", random_token())
 
+    if get_config(get_config_file(), "postgresql", "database_user") != "zulip" and need_secret(
+        "postgres_password"
+    ):
+        add_secret("postgres_password", random_token())
```

Comment:
This made me scratch my head for a bit -- since it's subtle, it's worth making sure makes it into the documentation clearly.  Specifically, using a database user that is not `zulip` means that the Unix username doesn't match up with the database username, so `ident` authorization doesn't work, so even databases on the same host need a password to authenticate.

Reply:
@alexmv where would you suggest adding this info in the docs?